corehq/apps/app_manager/static/app_manager/js/app_manager_media.js

```diff
@@ -187,4 +187,29 @@ hqDefine('app_manager/js/app_manager_media.js', function () {
         initNavMenuMedia: initNavMenuMedia,
         AppMenuMediaManager: AppMenuMediaManager
     };
+
+    function interpolatePath(path){
+        // app media attributes are interpolated on server side, media uploads should also be interpolated
+        // See corehq.apps.app_manager.views.media_utils.process_media_attribute
```

Comment:
The mobile requires it to be a URI and that is the format that was chosen for files: `jr://file`